src/magicbox/SuggestionsManager.ts

```diff
@@ -65,8 +65,8 @@ export class SuggestionsManager {
     let target = $$(<HTMLElement>e.target);
     let targetParents = target.parents(this.options.selectableClass);
 
-    //e.relatedTarget is not available if moving off the browser window
-    if (e.relatedTarget) {
+    //e.relatedTarget is not available if moving off the browser window or is an empty object `{}` when moving out of namespace in LockerService.
+    if (e.relatedTarget && e.relatedTarget.getAttribute != undefined) {
```

Comment:
I have a feeling this could be expressed better with some kind of helper class inside `Dom` module ?

An `Event` class, which would have a `hasValidRelatedTarget()` method, which could contain that comment/Locker service specific stuff.

Reply:
Do you think we should instead have a `isNullOrEmpty()` method ?